### ISSUE TITLE
Noob question 2 :  borg mount doesn't mount ?

### ISSUE THREAD

**Issue by PerfMonk** [borgbackup/borg#111]
I have followed installation from git for an ubuntu system and fuse and python3-llfuse are well installed but I can't mount 

I have tried to mount a repository and here are the results : 

(borg-env)root@m193543:~/git/borg# borg mount /media/trebe03/6f71065c-9172-418b-bf9f-f801f39ac37d/borg_bt /tmp/borg_bt
borg: loading fuse support failed [ImportError: No module named 'llfuse']
borg: Exiting with failure status due to previous errors

The FUSE packages are installed : 

(borg-env)root@m193543:~/git/borg# dpkg -l|grep fuse
ii  exfat-fuse                                  1.1.0-2                                    amd64        read and write exFAT driver for FUSE
ii  fuse                                        2.9.4-1ubuntu1                             amd64        Filesystem in Userspace
ii  gvfs-fuse                                   1.24.1-1ubuntu3                            amd64        userspace virtual filesystem - fuse server
ii  ifuse                                       1.1.2-0.1build2                            amd64        FUSE module for iPhone and iPod Touch devices
ii  libconfuse-common                           2.7-5                                      all          Common files for libConfuse
ii  libconfuse0:amd64                           2.7-5                                      amd64        Library for parsing configuration files
ii  libfuse2:amd64                              2.9.4-1ubuntu1                             amd64        Filesystem in Userspace (library)
ii  python-llfuse-doc                           0.40+dfsg-1                                all          Python bindings for the low-level FUSE API (documentation)
ii  python3-llfuse                              0.40+dfsg-1                                amd64        Python 3 bindings for the low-level FUSE API

Did I forgot something ?  Is that a problem with borg ?

Regards,
               Bernard

**Comment by ThomasWaldmann** [comment]
Hi, thanks for the feedback!

Yeah, the fuse mount stuff has some issues still and needs some tweaking before it works, we are currently trying to try the best way to solve and document it.

Some issues found:
- complains about permission denied on /etc/fuse.conf: on ubuntu this means your user is not in the "fuse" group. just add yourself there, log out and log in again and that will work.

**Comment by ThomasWaldmann** [comment]
fixed by:
https://github.com/borgbackup/borg/commit/859c33d42b846efa7b290007f6762d366fe4bf98

(in case there are more problems not fixed by this doc update, please reopen this)

**Comment by PerfMonk** [comment]
Thanks!  I added my user to the fuse group.  I will retry "borg mount" monday and give feedback.

**Comment by DrupaListo-com** [comment]
on linux ubuntu 12.04 LTS (codename precise) the "borg mount" is impossible to be made to work.

error I get is :
Loading fuse support failed [ImportError: No module named 'llfuse']

I am using:
- the borg prebuilt 32 bit binary directly downloaded and installed in $PATH
- ubuntu 12.04
- I have succesfully run borg 0.28.2 ( latest release as of now ) and backed up OK

But I cant mount the backup to "see" its all there and/or test restore selectively only one folder/file.

What I tried:
- I tried installing distro packages for llfuse - for both python 2.7 and 3.2 - seems like although there ARE packages for both 2 and 3, they are the same package actually - by insepcting the real package version  in aptitude
- I tried seeing where the borg binary searches for llfuse when mounting - via ldd and "readelf -d borg" but llfuse is not listed there
- I tried setting  up python3 virtualenv and installing llfuse via its own venv pip, no luck - "borg mount" does not run from the venv NOR globally

So now I have to either find where to symlink the venv package in some /usr/local/python3.2/site-packages type of dir OR inspect the borg source to see how is llfuse loaded and why does it fail on my system.

PS. Can llfuse be bundled INTO the borg standalone binary executable as other libs are ?
PS2. Should I try compiling my own borg from the release/git master ? Will that fix it for me ?
